codecov.yaml

```diff
@@ -1,7 +0,0 @@
-coverage:
-  status:
-    patch:
-      default:
-        target: auto
```

Comment:
According to the documentation this might help:
`target: 0%`

Reply:
It's the baseline of coverage that the commit must meet (Usually derived from master).

Reply:
Or even better: you can set a flag `ìnformational: true` This should make the CI pass no matter what.